admin/worker/runtime_health_check.go

```diff
@@ -0,0 +1,73 @@
+package worker
+
+import (
+	"context"
+	"fmt"
+
+	runtimev1 "github.com/rilldata/rill/proto/gen/rill/runtime/v1"
+	"go.uber.org/zap"
+	"golang.org/x/sync/errgroup"
+	"google.golang.org/protobuf/encoding/protojson"
+)
+
+const _allocatedRuntimesPageSize = 20
```

Comment:
Since this is just fetching relatively short host URLs, consider increasing this to maybe 500 or 1k